releases/unreleased/add-support-for-outputting-main-dependencies.yml

```diff
@@ -0,0 +1,6 @@
+---
+title: Add support for outputting main dependencies
+category: added
+author: Lewi Uberg <lewi@uberg.me>
+issue: null
+notes: add posibility to output only main dependencies
```

Comment:
Do you mind adding a proper note for the feature?
This would eventually go to the release notes and I feel they need to correct and precise.

Reply:
Good, but I have formatting comment. Will send the suggestion in the next iteration.
Thanks.